lib/std/crypto/25519/x25519.zig

```diff
@@ -19,8 +19,6 @@ pub const X25519 = struct {
     pub const public_length = 32;
     /// Length (in bytes) of the output of the DH function.
     pub const shared_length = 32;
-    /// Seed (for key pair creation) length in bytes.
-    pub const seed_length = 32;
```

Comment:
That shouldn't be removed.

Reply:
It's was moved to the keypair to unify things